### PR TITLE
chore(): update starters for angular 8 support

### PR DESCRIPTION
**Pull request by mhartington**
Update support for ng8 project base, as well as ionic/angular 4.7

### CODE REVIEW

File: angular/base/package.json [hunk 1]
```diff
@@ -13,46 +13,46 @@
...
-    "@angular-devkit/core": "~7.3.8",
-    "@angular-devkit/schematics": "~7.3.8",
-    "@angular/cli": "~7.3.8",
-    "@angular/compiler": "~7.2.2",
-    "@angular/compiler-cli": "~7.2.2",
-    "@angular/language-service": "~7.2.2",
-    "@ionic/angular-toolkit": "~1.5.1",
-    "@types/node": "~12.6.6",
-    "@types/jasmine": "~2.8.8",
+    "@angular-devkit/architect": "0.801.2",
```
Comment (dwieeb) [inline angular/base/package.json]: Are we sure we want these versions locked in this specifically?

File: angular/base/package.json [hunk 2]
```diff
@@ -13,46 +13,46 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "^7.2.2",
```
Comment (dwieeb) [inline angular/base/package.json]: These appear unalphabetized, which may result in unnecessary noise in version control when people `npm install`

File: angular/base/src/app/app.component.ts [hunk 3]
```diff
@@ -6,7 +6,8 @@ import { StatusBar } from '@ionic-native/status-bar/ngx';
 
 @Component({
   selector: 'app-root',
-  templateUrl: 'app.component.html'
+  templateUrl: 'app.component.html',
+  styleUrls: ['app.component.scss']
```
Comment (dwieeb) [inline angular/base/src/app/app.component.ts]: With this addition, the app.scss is no longer global, so the comment inside the file is incorrect.

@brandyscarney Didn't we have global.scss _and_ app.scss for a reason?

File: angular/official/blank/src/app/app-routing.module.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { PreloadAllModules, RouterModule, Routes } from '@angular/router';
 
 const routes: Routes = [
   { path: '', redirectTo: 'home', pathMatch: 'full' },
-  { path: 'home', loadChildren: './home/home.module#HomePageModule' },
+  { path: 'home', loadChildren: () => import('./home/home.module').then( m => m.HomePageModule)},
```
Comment (dwieeb) [inline angular/official/blank/src/app/app-routing.module.ts]: ❤️

File: angular/base/src/app/app.component.ts (hunk 3 above)
Comment (brandyscarney) [reply angular/base/src/app/app.component.ts]: Yes I believe this is why we were separating out the `app.scss` file for global styles/mixins.

File: angular/base/package.json [hunk 5]
```diff
@@ -13,46 +13,46 @@
...
-    "@angular/compiler-cli": "~7.2.2",
-    "@angular/language-service": "~7.2.2",
-    "@ionic/angular-toolkit": "~1.5.1",
-    "@types/node": "~12.6.6",
-    "@types/jasmine": "~2.8.8",
+    "@angular-devkit/architect": "0.801.2",
+    "@angular-devkit/build-angular": "0.801.2",
+    "@angular-devkit/core": "8.1.2",
+    "@angular-devkit/schematics": "8.1.2",
+    "@angular/cli": "8.1.2",
```
Comment (brandyscarney) [inline angular/base/package.json]: Curious why in other dependencies we do `~` but in these we don't?

File: angular/base/src/app/app.component.ts (hunk 3 above)
Comment (mhartington) [reply angular/base/src/app/app.component.ts]: So the logic here is that global should be for global styles and mixins, and app.component.scss should be scoped styles for the app component (if any is needed)


If we have global.scss and app.scss, they both are treated at the same level. Which doesn't really make sense to keep both. I'd say just keep global.scss and then if devs want to have more, they can be the one to make that call.

**Pull request merged**